pylintrc

```diff
@@ -64,21 +64,3 @@ max-branches=20
 
 [SIMILARITIES]
 min-similarity-lines=10
-
-[BASIC]
-# Naming hints based on PEP 8 (https://www.python.org/dev/peps/pep-0008/#naming-conventions).
-# Consider these guidelines and not hard rules. Read PEP 8 for more details.
-
-# The invalid-name checker must be **enabled** for these hints to be used.
-include-naming-hint=yes
-
-module-name-hint=lowercase (keep short; underscores are discouraged)
-const-name-hint=UPPER_CASE_WITH_UNDERSCORES
-class-name-hint=CapitalizedWords
-class-attribute-name-hint=lower_case_with_underscores
-attr-name-hint=lower_case_with_underscores
-method-name-hint=lower_case_with_underscores
-function-name-hint=lower_case_with_underscores
-argument-name-hint=lower_case_with_underscores
-variable-name-hint=lower_case_with_underscores
-inlinevar-name-hint=lower_case_with_underscores (short is OK)
```

Comment:
These options changes to xxxx-style, see https://pylint.pycqa.org/en/latest/user_guide/configuration/all-options.html#basic-checker.
The default value is enough, for example, `module-naming-style` is `snake_case`.

PS:
If the comment is true, these xxx-hint config never takes effect.
> The invalid-name checker must be **enabled** for these hints to be used.

Reply:
This is very nice investigation and makes the context much clearer.